data/blog/8th-wave-of-bitcoin-grants.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: "Eighth Wave of Bitcoin Grants"
+date: '2024-10-23'
+tags: ['OpenSats', 'grants', 'bitcoin', 'wave']
+draft: false
+authors: ['arvin', 'dergigi']
+images: ['/static/images/blog/59-bitcoin-8th-wave.jpg']
+summary: "OpenSats is funding 4 more open-source projects in the bitcoin ecosystem."
+---
+
+OpenSats is proud to announce its eighth wave of Bitcoin grants, dedicated to
+supporting innovative open-source projects that enhance the growth and
+resilience of the Bitcoin ecosystem.
+
+We remain committed to empowering developers worldwide who are building
+groundbreaking tools, improving user experience, and strengthening the overall
+security of Bitcoin and its users.
+
+This wave awards grants to four outstanding projects, each playing a vital role
+in advancing various aspects of the Bitcoin network. In no particular order, the
+recipients are:
+
+- [Citadel-Tech](#citadel-tech)
+
+- [Lampo](#lampo)
+
+- [Cashu Nutshell](#cashu-nutshell)
+
+- [PickhardtPayments Plugin](#pickhardtpayments-plugin)
+
+These grants are sourced from our General Fund, sustained by the generosity of
+donors who share our vision for a thriving, decentralized future. If you would
+like to support open-source projects that help Bitcoin flourish, please consider
+donating to the General Fund:
+
+<center>
+    <DonateToGeneralFundButton/>
+</center>
+
+Let's take a closer look at these projects to discover their purpose, impact,
+and the value they bring to the Bitcoin ecosystem..
+
+---
+
+### Citadel-Tech
+
+Citadel-Tech is dedicated to advancing privacy within the Bitcoin ecosystem
+through the development of an atomic swap protocol first proposed by Greg
+Maxwell in 2013. By implementing the
+[Maxwell-Belcher](https://gist.github.com/chris-belcher/9144bd57a91c194e332fb5ca371d0964)
+atomic swap protocol, the project enables truly private ownership transfers of
+Bitcoin UTXOs between participants. With a focus on providing "passive privacy,"
+even for non-users, this project ensures that swaps are indistinguishable from
+standard Taproot transactions today.
+
+Repository: [citadel-tech](https://github.com/citadel-tech/coinswap)  
+License: Apache-2.0 & MIT
+
+### Lampo
+
+Lampo is an open-source project that aims to build a modular and customizable
+lightning node implementation for the Bitcoin ecosystem. Designed to integrate
+with the Lightning Development Kit (LDK) and the LNprototest test suite, Lampo
+offers flexibility and innovation through features like a swappable API, wallet,
+on-chain data source, and a plugin system similar to Core Lightning. Lampo is
+positioned as a fully open-source alternative to company-driven lightning
+implementations, promoting greater interoperability, customization, and
+experimentation within the Bitcoin and lightning communities.
```

Comment:
Capitalize _lightning_ so that it reads "...Bitcoin and Lightning communities"

Reply:
Good one. Will do.